djangocms_admin_style/sass/components/_tables.scss

```
@@ -292,6 +292,8 @@
     }
     .paginator {
         box-shadow: none !important;
+        padding-left: 10px;
```

```suggestion
        padding-inline-start: 10px;
```
With pseudo properties we can automatically support RTL languages.